README.md

```diff
@@ -112,7 +112,9 @@ Optional:
 * `extra_organisation_slugs` — additional organisations which own this site. Used for access control in Transition.
 * `homepage_furl` — friendly URL displayed on 404/410 pages. Should redirect to the `homepage`. Doesn't need to include 'http' or 'https'.
 * `aliases` — list of alias domains
-* `global` — set a global redirect or archive for all paths
+* `global` — set a global redirect or archive for all paths. There are two expected values for this:
+    - `=401` - this is a global archive, all site URLs will show a page saying the site has been archived
```

Comment:
This should be 410

Reply:
Oops, yep... thanks! Fixed that in 595fd704578339e548ffe47bfa0f676b7319011f.